### DISCUSSION TITLE
Copy link function failed

### DISCUSSION THREAD

**Discussion by betheboy** [docmost/docmost#266] [Help / Questions]
when i click to copy the page link,there comes a link copied windows.
![image](https://github.com/user-attachments/assets/56ce68ec-4f97-43df-9d9f-29d01409a38c)

![image](https://github.com/user-attachments/assets/cdf4cb0e-dfa0-4f1e-8dbf-1b8a668d94d9)

But when i was trying to paste the copied link, it shows nothing!

How to solve this problem,big thanks!

**Comment by Philipinho** [comment]
Do you have SSL setup?

**Reply by betheboy** [reply]
NO, I do not have SSL setup.

**Comment by fengdu126** [comment]
must be ssl then coped
